lms/static/js/learner_dashboard/views/program_details_sidebar_view.js

```diff
@@ -81,6 +91,17 @@ class ProgramDetailsSidebarView extends Backbone.View {
 
     return type.replace(/\s+/g, '-');
   }
+
+  trackPathwayClicked(event) {
+    var button = event.currentTarget;
+    window.analytics.track('edx.bi.dashboard.program.pathway.clicked', {
+      category: 'pathways',
+      program_uuid: this.programModel.attributes.uuid,
```

Comment:
It might be worth considering which version of the uuid we get here - hex or formal. (i.e. no-dashes or dashes)

I don't have a suggestion on which makes sense. But just wanted to make sure whichever way was intentional. (like, which do we use on other events? which did we use on credentials?)

Reply:
You use hex in credentials. Would love to chat with you about. Lots of issues with that for me.